soh/src/code/z_play.c

```diff
@@ -1996,4 +1996,5 @@ void Gameplay_PerformSave(GlobalContext* globalCtx) {
     } else {
         Save_SaveFile();
     }
+    Overlay_DisplayText(3.0f, "Game Saved");
```

Comment:
This will put overlay text whenever the game is saved, even if auto save is off. I'm debating on whether or not we want that. 🤔

Reply:
I decided that would be good to have for exact timing reasons, given how long it takes to close the pause menu, the lack of lag that console players might not be accounting for, and the fact that we now have a PR for whether or not to continue playing after you manually save, knowing that you're DEFINITELY good to exit/restart would be useful.  It will be trivial to remove on manual saves though, if desired.

Reply:
i'd prefer to not have the text, but i can see the argument for having it on even with manual saves...

i feel like having the text as an option separate from autosave might make sense, but i don't like suggesting *yet another* menu option

Reply:
Given that I'm already using hidden cvars for autosave, it would also be super easy to just make another one or two for showing the text after manual and autosaves (and if the powers that be determine that having the text for manual saving needs to be controlled separately from the text for autosaving, I'd have the hidden cvar for the manual save text off by default).